### DISCUSSION TITLE
Does this support other types of ACLs?

### DISCUSSION THREAD

**Discussion by edthezombie** [vladimirs-git/cisco-acl#3] [Announcements]
Love this so far. Just wondering, does this support standard ACL's and IPv6? It seems to only pickup IPv4 extended access-lists. Thanks in advance.

**Comment by vladimirs-git** [comment]
Great to know you're enjoying it!
While Standard ACL is supported, but it is implemented in a somewhat unconventional manner. In my experience, there have been no projects related to standard ACL to refine this package.
I don't plan to implement the new IPv6 feature in the near future, for the same reason, there are no projects requiring IPv6.

Example of a standard ACL.
```python
import cisco_acl

config = """
ip access-list standard ACL
  permit 10.0.0.0 0.0.0.255
  permit 10.0.1.0 0.0.0.255
  permit 10.0.2.0 0.0.0.255
"""
acl = cisco_acl.Acl(line=config, type="standard")
print(acl)
# ip access-list standard ACL
#   permit 10.0.0.0 0.0.0.255
#   permit 10.0.1.0 0.0.0.255
#   permit 10.0.2.0 0.0.0.255

for ace in acl.items:
    print(f"{ace.srcaddr.prefix=}")
# ace.srcaddr.prefix='10.0.0.0/24'
# ace.srcaddr.prefix='10.0.1.0/24'
# ace.srcaddr.prefix='10.0.2.0/24'
```
